tofu/harvester/modules/vm/main.tf

```diff
@@ -4,7 +4,7 @@ locals {
   temp_node_names = flatten([
     for node_group in var.nodes : [
       for i in range(node_group.count) : {
-        name =  "${var.hostname_prefix}-${join("-", node_group.role)}-${i}"
+        name =  "${join("-", node_group.role)}-${i}"
```

Comment:
Will this not have the name specified in variables prefixed to it anymore?

Reply:
Sure I'm fine with this being in the overarching "improve the things" task. For now the consistency is good.